static/js/deprecated_feature_notice.js

```diff
@@ -0,0 +1,8 @@
+import $ from "jquery";
+
+import render_deprecated_feature_notice from "../templates/deprecated_feature_notice.hbs";
+
+export function initialize() {
+    const rendered_deprecated_feature_notice = render_deprecated_feature_notice();
+    $(".app").append(rendered_deprecated_feature_notice);
```

Comment:
Same comment as https://github.com/zulip/zulip/pull/18835/files#r651258932; this is the most straightforward migration, but it also seems clear that a next commit should move the `ui.js` code for this feature to this file at the very least, and possibly we should make it render when used rather than during page initialization.

i.e. the `$("#deprecation-notice-message").text(message);` should become passing `message` as a parameter to the `render` method.